widget/embedded/src/components/ChainsChip/ChainsChip.tsx

```diff
@@ -0,0 +1,15 @@
+import type { PropTypes } from './ChainsChip.types';
+import type { PropsWithChildren } from 'react';
+
+import React from 'react';
+
+import { Chip } from './ChainsChip.styles';
+
+export function ChainsChip(props: PropsWithChildren<PropTypes>) {
```

Comment:
As we decided to use `Blockchain` instead of `Chain` for consistency, Please rename the component to `BlockchainChip`